### PR TITLE
fix: map container template ref not found

### PR DESCRIPTION
**Pull request by prashis15**

### CODE REVIEW

File: src/components/map/mixins/withPrivateMethods.ts [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import Vue from 'vue';
+import { ref } from '@vue/composition-api';
+
 export default Vue.extend({
+  setup(_, context) {
+    const templateRefs = ref(context.refs);
```
Comment (vinayakkulkarni) [inline src/components/map/mixins/withPrivateMethods.ts:6]:
```suggestion
    const templateRefs: Ref<Element> = ref(context.refs);
```

File: src/components/map/mixins/withPrivateMethods.ts [hunk 2]
```diff
@@ -1,5 +1,11 @@
 import Vue from 'vue';
+import { ref } from '@vue/composition-api';
```
Comment (vinayakkulkarni) [inline src/components/map/mixins/withPrivateMethods.ts:2]:
```suggestion
import { ref } from '@vue/composition-api';
import type { Ref } from '@vue/composition-api';
```

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]

File: src/components/map/mixins/withPrivateMethods.ts (hunk 1 above)
Comment (prashis15) [reply src/components/map/mixins/withPrivateMethods.ts]: added typings for it👍🏼

**Review by vinayakkulkarni — APPROVED** [review]
LGTM

**Pull request merged**